src/mlpack/methods/ann/layer/serialization.hpp

```diff
@@ -67,6 +67,7 @@
     CEREAL_REGISTER_TYPE(mlpack::PaddingType<__VA_ARGS__>); \
     CEREAL_REGISTER_TYPE(mlpack::RBFType<__VA_ARGS__>); \
     CEREAL_REGISTER_TYPE(mlpack::SoftmaxType<__VA_ARGS__>); \
+    CEREAL_REGISTER_TYPE(mlpack::SoftminType<__VA_ARGS__>); \
```

Comment:
@AdarshSantoria did we forget to do this for the PReLU layer?  If so, want to open a simple PR to fix it?

Reply:
No, I had added this for PReLU in #3420.

Reply:
Weird, the diff here doesn't show it. Anyway, you are right, it is added correctly to `master`. Thanks :)